docs/iris/src/whatsnew/contributions_2.0/bugfix_2017-Oct-24_area-weights-speedup.txt

```diff
@@ -0,0 +1 @@
+ * Upto 3x speedup when using regrid_area_weighted_rectilinear_src_and_grid with non-masked data. (#2730)
```

Comment:
The name of this file refers to it as a bugfix, but 'enhancement' would be more approriate.